src/map.react.js

```diff
@@ -278,10 +282,6 @@ export default class MapGL extends Component {
     return 'inherit';
   }
 
-  _getMap() {
```

Comment:
Many applications have come to depend on this to get to the internal map instance. We should add a console warning to this for a couple of versions before deprecating it completely. Can we add this back with a console warning? Something like "you can access the `_map` object directly and this method will eventually be deprecated" will do.